src/toil/batchSystems/singleMachine.py

```diff
@@ -120,6 +120,33 @@ def __init__(self, config, maxCores, maxMemory, maxDisk):
             self.workerThreads.append(worker)
             worker.start()
 
+    def _get_debug_cmd(self, jobCommand):
```

Comment:
I dislike camelCase in Python. But I dislike inconsistency even more.

On Thu, Jul 14, 2016 at 10:18 AM, Brad Chapman <notifications@github.com>
wrote:

> In src/toil/batchSystems/singleMachine.py
> <https://github.com/BD2KGenomics/toil/pull/1048#discussion_r70845448>:
>
> > @@ -120,6 +120,33 @@ def __init__(self, config, maxCores, maxMemory, maxDisk):
> >              self.workerThreads.append(worker)
> >              worker.start()
> >
> > +    def _get_debug_cmd(self, jobCommand):
>
> Thanks Hannes, fixed. Apologies, it's quite hard to get used to camelCase
> when you normally use the pep8 styles.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/BD2KGenomics/toil/pull/1048/files/d71a123c4a03b56ef322cdeb0270a847bf83f6bc#r70845448>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AE562MZN5BXY41uAREp6dfnJeLRzShJoks5qVm9QgaJpZM4JG9ej>
> .
>



-- 
Hannes Schmidt
Software Engineer
Center for Biomolecular Science and Engineering
University of California, Santa Cruz

(206) 696-2316 (cell)
hannes@ucsc.edu